### PR TITLE
fix: Freeze 'Series column' on detailed results table (11800)

### PR DESCRIPTION
**Pull request by Arun-chaitanya**
## Problem
Freeze 'Series column' on detailed results table (#11800)

## Changes
Added sticky feature to the table and enabled sticky prop for series column

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by Twixes — CHANGES_REQUESTED** [review]
This is very cool! There is just one major issue: stickiness is wonky if more than one column is sticky, see below. 

The checkbox and series name should be sticky _together_. In this case this can be fixed easily by combining those two columns – there are no column alignment considerations because checkboxes are always the same width. However, the API should not allow this situation. I think it may be better to do things this way: remove the `sticky` param from columns, and instead add a `LemonTable` prop called `firstColumnSticky`.

Also, it would be worth having a border-right on the sticky column when scrolled, to make the stickiness clear interface-wise.

**Comment by Twixes** [comment]
(Also, merge your branch with PostHog/posthog master, as we've fixed an issue with checks failing on external PRs)

File: frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss [hunk 1]
```diff
@@ -177,3 +177,15 @@
     border-top: 1px solid var(--border);
     cursor: default;
 }
+
+.LemonTable__header--sticky,
+.LemonTable__td--sticky {
```
[the remark this answers is not in the document]
Comment (Arun-chaitanya) [reply frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss:182]: Sure. Will do the required changes and keep the conversation going.

**Comment by Arun-chaitanya** [comment]
@Twixes, I made all the changes as recommended. Please check.

**Comment by Arun-chaitanya** [comment]
I messed up with this PR, so closing this and opening a PR (https://github.com/PostHog/posthog/pull/16825). @Twixes I did all the changes recommended by you in that PR.

**Pull request closed without merging**